analytics/service_test.go

```diff
@@ -17,11 +25,352 @@ func Test_extractIntParam(t *testing.T) {
 		So(err, ShouldBeNil)
 
 		q := r.URL.Query()
+		// using: https://jwt.io/#debugger-io
+		// the q.Set() below has:
+		// HEADER:ALGORITHM & TOKEN TYPE
+		/*
```

Comment:
I've been asked in the past not to comment out code, so perhaps you should remove this, unless its been commented out for a particular reason I've not spotted?